### ISSUE TITLE
wrong on Train on coco from scratch(not necessary)

### ISSUE THREAD

**Issue by Alan-D-Chen** [zylo117/Yet-Another-EfficientDet-Pytorch#616]
I want to Train on coco from scratch.
I get that: 
compound_coef =  1 ； num_workers = 12 ； batch_size = 4 ；lr = 1e-4 in train.py and  num_gpus: 2    in coco.yml.
but I get this thing:
`loading annotations into memory...`
`Done (t=15.42s)`
`creating index...`
`index created!`
`loading annotations into memory...`
`Done (t=0.51s)`
`creating index...`
`index created!`
`[Info] initializing weights...`
`Step: 499. Epoch: 0/500. Iteration: 500/29571. Cls loss: 1.84222. Reg loss: 11.28150. Total loss: 13.12371:   2%|  | 499/29571 [07:27<6:49:31,  1.18it/s]checkpoint...
Step: 999. Epoch: 0/500. Iteration: 1000/29571. Cls loss: 1.33001. Reg loss: 7.30902. Total loss: 8.63903:   3%|   | 999/29571 [14:48<6:44:04,  1.18it/s]checkpoint...
Step: 1499. Epoch: 0/500. Iteration: 1500/29571. Cls loss: 1.28000. Reg loss: 6.70479. Total loss: 7.98479:   5%| | 1499/29571 [22:10<6:34:25,  1.19it/s]checkpoint...
Step: 1999. Epoch: 0/500. Iteration: 2000/29571. Cls loss: 1.24906. Reg loss: 6.53883. Total loss: 7.78789:   7%| | 1999/29571 [29:31<6:36:56,  1.16it/s]checkpoint...
Step: 2499. Epoch: 0/500. Iteration: 2500/29571. Cls loss: 1.26784. Reg loss: 5.32663. Total loss: 6.59447:   8%| | 2499/29571 [37:01<10:05:26,  1.34s/itcheckpoint...
Step: 2999. Epoch: 0/500. Iteration: 3000/29571. Cls loss: 1.20817. Reg loss: 5.71568. Total loss: 6.92385:  10%| | 2999/29571 [44:28<6:30:40,  1.13it/s]checkpoint...
Step: 3499. Epoch: 0/500. Iteration: 3500/29571. Cls loss: 1.21015. Reg loss: 5.53784. Total loss: 6.74799:  12%| | 3499/29571 [51:56<6:09:06,  1.18it/s]checkpoint...
Step: 3999. Epoch: 0/500. Iteration: 4000/29571. Cls loss: 1.18565. Reg loss: 5.76265. Total loss: 6.94830:  14%|▏| 3999/29571 [59:36<6:14:06,  1.14it/s]checkpoint...
Step: 4499. Epoch: 0/500. Iteration: 4500/29571. Cls loss: 1.20720. Reg loss: 5.47375. Total loss: 6.68095:  15%|▏| 4499/29571 [1:07:08<6:13:27,  1.12it/checkpoint...
Step: 4999. Epoch: 0/500. Iteration: 5000/29571. Cls loss: 1.13282. Reg loss: 5.85982. Total loss: 6.99264:  17%|▏| 4999/29571 [1:14:34<5:54:57,  1.15it/checkpoint...
Step: 5499. Epoch: 0/500. Iteration: 5500/29571. Cls loss: 1.13337. Reg loss: 5.28262. Total loss: 6.41599:  19%|▏| 5499/29571 [1:21:57<5:47:27,  1.15it/checkpoint...
Step: 5999. Epoch: 0/500. Iteration: 6000/29571. Cls loss: 1.16179. Reg loss: 5.45389. Total loss: 6.61567:  20%|▏| 5999/29571 [1:29:30<5:45:52,  1.14it/checkpoint...
Step: 6499. Epoch: 0/500. Iteration: 6500/29571. Cls loss: 2.16591. Reg loss: 3.46935. Total loss: 5.63526:  22%|▏| 6499/29571 [1:37:01<5:42:17,  1.12it/checkpoint...
Step: 6628. Epoch: 0/500. Iteration: 6629/29571. Cls loss: 1.18831. Reg loss: 6.09740. Total loss: 7.28571:  22%|▏| 6629/29571 [1:38:56<5:40:0`
this is pretty weird~.
Is right that `[Info] initializing weights` takes 5 or 6 hours ?

**Comment by Alan-D-Chen** [comment]
@zylo117 please give me a hand~, is it OK to initialize weights for 5 or 6 hours?
And I have read your words from "Train on coco from scratch(not necessary)", is it really to  spend months on training on coco from scratch?
